src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Incentive.kt

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders
+
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.IEPLevel
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Incentive
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.IncentiveId
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.OffenderBooking
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.ReferenceCode
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.repository.IncentiveRepository
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.repository.ReferenceCodeRepository
+import java.time.LocalDateTime
+
+@DslMarker
+annotation class NewIncentiveDslMarker
+
+@NomisDataDslMarker
+interface NewIncentiveDsl
+
+@Component
+class NewIncentiveBuilderRepository(
```

Comment:
Modelled incentives in the new DSL style. As we already have an IncentiveBuilder and DSLMarker etc. I chose to prefix everything with `New` to indicate its using the new DSL. Everything can be renamed once the old DSL has been replaced.